style.css

```diff
@@ -0,0 +1,3 @@
+div {
+  color: blue;
+}
```

Comment:
- [ ] [OPTIONAL] Goob job adding styles to your project 🐎, using `div` as a selector is going to change all of the styles of your <div> tags in your project, in this case it works because you have just one `<div>` tag, but in bigger projects, the most probable it is not going to work 🤯, I suggest you use a `class` or `id` to customize in a better way your styles 💯.